src/platform/forms/sass/_m-review-box-field.scss

```diff
@@ -0,0 +1,42 @@
+.review-box {
```

Comment:
Can this Sass be scoped in the application directory? Ideally we want to isolate these kinds of things

Reply:
Okay. Should I also be moving ReviewBoxField.jsx to be in the application directory?

Reply:
If you can, yeah. I'm not super familiar with the forms system (cc @cvalarida), but we try and scope all the code for an individual app within that app if possible

Reply:
Okay. I think this field may eventually be used other places, but we can cross that bridge when we get there.